src/buttonCall.js

```diff
@@ -0,0 +1,29 @@
+import _ from 'lodash'
+export default function(buttonType) {
+  var success = true;
+  var self = this;
+  if (this.props[buttonType]) {
+
+    // If[buttonType] is not a function, then close modal if true
+    if (!_.isFunction(this.props[buttonType])) {
+      if (this.props[buttonType] === true) {
+        this.props.onHide();
```

Comment:
do you think it could be dangerous to implicitly assume the context in which this is being called has onHide available?